### DISCUSSION TITLE
Version 6.0.1.0

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#762] [Announcements]
# Version 6.0.1.0
## New features
  * **AltOption** added to basic tone group. This will make any parameter adjustments act as if the alt or option key were held down. Adobe Lightroom shows special displays in this situation for some adjustments, such as showing clipping.
  * New commands:
    * CurveRefineSaturation
    * Point curve blacks or highlights
    * Mask new: subject, sky, background, object, people
    * Mask add: subject, sky, background, object, people
    * Mask subtract: subject, sky, background, object, people
    * Mask intersect: subject, sky, background, object, people
## Improvements
  * Experimental **local_Whites2012** and **local_Blacks2012** now supported officially by Lightroom as **local_Whites** and **local_Blacks**. Profiles with older version of names will still work as they are automatically updated when loaded. Recommend resaving any such profiles after they've been loaded to save with new names.
  * Supports Process Version 6.
  * Slider increase/decrease now works for library view as well as develop.
  * Limits can only be set in Develop module and did not show in other modules when options dialog opened. Now a message is informing the user of the reason.
  * Display when pickup fails (controller and Lightroom are out of sync) now explains why you are seeing two values.
## Bug fixes
  * Improved app stability.
  * Fixed error when closing Lightroom while app has profile to save.
  * More resilient to unannounced changes in parameter handling in Lightroom.
  * Masking toggle fixed.
  * Error when profile assigned to masking fixed. (6.0.0.0 withdrawn for bug to be fixed in this point release.)
  * Translations updated. MIDI2LR uses Adobe strings for many of the translations, and Adobe sometimes changes the key for finding the string, breaking the translation in MIDI2LR.
  * Pressing the Alt (Option) now will change the display while moving a slider in MIDI2LR. The Alt (Option) key is used with several different sliders to show an optional display that highlights what is being changed. This was fixed by Adobe, who addressed the Lightroom bug causing it to previously ignore the key while using MIDI2LR.

Note for हिन्दी:- इंस्टॉलर हिंदी का समर्थन नहीं करता है। हालांकि, आवेदन करता है। MIDI2LR इंस्टॉलर के लिए उपयोग की जाने वाली भाषा की परवाह किए बिना सभी भाषाओं का समर्थन करता है।

<hr /><em>This discussion was created from the release <a href='https://github.com/rsjaffe/MIDI2LR/releases/tag/'>Version 6.0.1.0</a>.</em>

**Comment by Masterori** [comment]
Thanks for coming back to the project and updating it.

There is a bug when using the stream deck plus dials where the settings keep repeating every 2 seconds until you change modules, would you be able to look into that next please?

**Reply by rsjaffe** [reply]
You should open that as a separate discussion to be addressed there, not here. It's more likely your setup than it is a bug.

**Reply by Masterori** [reply]
Someone has already made a discussion on this [This Discussion](https://github.com/rsjaffe/MIDI2LR/discussions/727) as far I can tell there isn't a good conclusion to it being solved

**Reply by rsjaffe** [reply]
let's continue the discussion there.

**Comment by cstsang** [comment]
OMG, Welcome back!

**Comment by Darek24Szubi** [comment]
I have this problem.

After installing the new version I have very often a message for example Exposure controller not synchronised
Previously I did not have such a problem.

**Reply by psylapse** [reply]
Hi,

I also have the same problem. I did an uninstall/reinstall of Midi2LR, but the problem persists.  The message disappears when you pass the control through the zero point, so that the setting then follows the control correctly.  However switch to another control and then back again and you have to go through the same procedure again.  This would not be a problem except that if you return to an image to edit further, the previous setting is of course wiped as soon as you apply this "fix";.  Notwithstanding I am very grateful to the author for all of his efforts.  After all, it seems probable that this problem has arrived through some undocumented change in Lightroom itself

**Reply by psylapse** [reply]
OK, solved the problem for me, though I dont understand why. I unchecked pickup mode (in settings) and the problem has gone away

**Reply by Darek24Szubi** [reply]
Where exactly did you make this change?

**Reply by psylapse** [reply]
Open the main panel of Midi2LR, click on settings button near top right and then disable pickup (first setting listed)

**Reply by Darek24Szubi** [reply]
Thank you! I think it actually works!

**Reply by Darek24Szubi** [reply]
Seemingly it is ok because the messages do not appear. Unfortunately, adjusting the dials is sometimes completely out of control. The settings jump by strange values.
I reinstalled the program, it did not help.
I believe that this problem will be fixed in the next version.

**Reply by AndyMcFishka** [reply]
Hey guys! I fixed that issue in this PR: https://github.com/rsjaffe/MIDI2LR/pull/771

**Reply by rsjaffe** [reply]
Version 6.0.2.0 uses that fix. Thanks!

**Comment by antzpantz** [comment]
Could you try uninstalling the previous version and reinstalling the latest
version and try again?

**Reply by Darek24Szubi** [reply]
Unfortunately, I did the installation by upgrading the version. 
Do you think it's worth trying to uninstall the plugin and then do a fresh install of the latest version once again?

**Comment by antzpantz** [comment]
Of course it's worth trying. Just export all of your settings and restore
them after reinstallation.

**Reply by Darek24Szubi** [reply]
IN THE lIGHTROOM IN THE MODULE OF THE LABELS, the ODINSTALLATION OPTION IS INACTIVE.

what would be the best way to uninstall the plug-in? Would physically removing it be correct?

**Comment by spacewhippet** [comment]
Thank you so much for the update!!

**Comment by Papp-Anti** [comment]
I need help! I just bought an X touch compact midi controller and added the midi2lr 6.0.1.0 plugin to the lr 12.2.1 version and the lr slowed down terribly. 2 minutes to export an image, which used to be 1-2 seconds. It also follows the movement of the faders with a delay, and even if I move several faders at the same time, it freezes for minutes. The pc is a Ryzen7 5700x 32GB ram and a win10 system. How could this slowdown be eliminated?
